common/channels.go

```diff
@@ -7,3 +7,15 @@ func GetClosedUnbufferedChannel() chan struct{} {
 
 	return ch
 }
+
+// ErrFromChan will get the error from channel
+func ErrFromChan(errChan chan error) error {
+	for {
```

Comment:
The `for` is useless, because this func will return either on err or on default.

Reply:
ahh, right :+1: